spec/rubocop/ast/traversal_spec.rb

```diff
@@ -64,6 +64,30 @@ def baz(&block)
     end
   end
 
+  context 'when a class defines `on_itblock`', :ruby34, broken_on: :parser do
+    let(:klass) do
+      Class.new do
+        attr_reader :calls
+
+        include RuboCop::AST::Traversal
+        def on_itblock(node)
+          (@calls ||= []) << node.children.first&.type
```

Comment:
Safe navigation seems unnecessary for this test cases.

```suggestion
          (@calls ||= []) << node.children.first.type
```

Reply:
I removed it